renderdoc/driver/gl/wrappers/gl_buffer_funcs.cpp

```diff
@@ -2960,6 +2960,25 @@ bool WrappedOpenGL::Serialise_glVertexArrayVertexAttribOffsetEXT(GLuint vaobj, G
   return true;
 }
 
+std::function<void()> WrappedOpenGL::ConvertClientMemoryPointer(GLuint index, const void *&pointer)
+{
+  GLuint buffer;
+  m_Real.glGetIntegerv(eGL_VERTEX_ARRAY_BINDING, (GLint *)&buffer);
```

Comment:
Ah that was a mistake, it was supposed to be GL_VERTEX_ATTRIB_ARRAY_BUFFER_BINDING, which Im doing now in my latest version in CopyClientMemoryArrays.